calliar_server/server/views.py

```diff
@@ -27,29 +32,34 @@ def render_to_template(self):
     
     def get(self, request, *args, **kwargs):
         return self.render_to_template()
-        
+
+    def save_image(self, data, save_path):
+        b = data.encode()
+        z = b[b.find(b'/9'):]
+        Image.open(io.BytesIO(base64.b64decode(z))).save(save_path)
+           
     def post(self, request, *args, **kwargs):
-        try:
-            data = json.loads(request.body)
-            file_name = data['newImageName'].split('.')[0]
-            ctr = '' 
-            while True:
-                if ctr+file_name+'.jpg' not in os.listdir('server/static/processed_images'):
+        data = json.loads(request.body)
+        file_name = data['newImageName'].split('.')[0]
+        existOnServer = bool(data['existOnServer'])
+        ctr = '' 
+        while True:
+            if ctr+file_name+'.jpg' not in os.listdir('server/static/processed_images'):
+                if existOnServer:
                     shutil.move(f"server/static/images/{data['oldImageName']}",
                                 f"server/static/processed_images/{ctr+file_name}.jpg")
-                    break 
                 else:
-                    if ctr == '':
-                        ctr = '1'
-                    else:
-                        ctr = str(int(ctr)+1)
-            print(file_name)
-            file_path = f"server/static/data/{ctr+file_name}.json"
-            json.dump(data['sketch'],open(file_path, 'w'))
-            result = JsonResponse({'result':True})
-        except Exception as e:
-            print(e)
-            result = JsonResponse({'result':False})
+                    self.save_image(data['imageBlob'], f"server/static/processed_images/{ctr+file_name}.jpg")
```

Comment:
I think this funciton is not used anywere else. Can we just put it here ike this:
```python
image_bin = data['imageBlob'].encode()
save_path = f"server/static/processed_images/{ctr+file_name}.jpg"
image_object = io.BytesIO(base64.b64decode(image_bin[image_bin.find(b'/9')]))
image_object.save(save_path)
```
What do you think?